Resources/Prototypes/Entities/Objects/Devices/Electronics/door_access.yml

```diff
@@ -179,6 +179,14 @@
   - type: AccessReader
     access: [["Research"], ["Medical"]]
 
+- type: entity
```

Comment:
these need to go in _DV file and the science one needs science replaced with epistemics
also add a Surgery/Robotics one